src/service/migrations/eth-to-keys.go

```diff
@@ -0,0 +1,75 @@
+package migrations
+
+import (
+	"context"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/pkg/log"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/service/errors"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/service/formatters"
+	"github.com/consensys/quorum-hashicorp-vault-plugin/src/utils"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func (c *controller) NewEthereumToKeysOperation() *framework.PathOperation {
+	return &framework.PathOperation{
+		Callback:    c.ethToKeysHandler(),
+		Summary:     "Migrates the current Ethereum accounts to the keys namespace",
```

Comment:
Minor: I think you can skip "the current" from the summary as it is self-explanatory already your are moving Ethereum Accounts

Reply:
done